src/pages/home/Home.jsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from "react";
+import { instance } from "../../api/api";
+import {
+  Container,
+  Banner,
+  Intro,
+  List,
+  Logo,
+  Item,
+  Thum,
+  Info,
+  Name,
+  Price,
+} from "./styles";
+
+const Items = ({ props }) => {
+  return (
+    <Item>
+      <Thum src={props.thumbnail} alt="상품이미지" />
+      <Info>
+        <Name>{props.title}</Name>
+        <Price>{props.price.toLocaleString()}</Price>
+      </Info>
+    </Item>
+  );
+};
+
+const Home = () => {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await instance.request("/products", {
+          method: "get",
+          headers: {
+            masterKey: true,
+          },
+        });
+        if (res.status === 200) {
+          setData(res.data);
+          console.log(res.data);
+        }
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <Container>
+      <Banner>
+        <Logo>Vin-co</Logo>
+      </Banner>
+      <div>
+        <Intro>Vin-co를 방문해주셔서 감사합니다.</Intro>
+        <Intro>당신이 찾는 골동품 모두 여기에!</Intro>
+      </div>
+      <List>
+        {data.map((api) => (
+          <Items key={api.id} props={api} />
+        ))}
```

Comment:
```suggestion
        {products.map((product) => (
          <Items key={products.id} props={products} />
        ))}
```

위에서 state 명을 좀 더 명확하게 바꾸면 이런식으로 사용할 수 있을 것 같습니다!